src/jetpack-editor-setup.js

```diff
@@ -0,0 +1,49 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import { JETPACK_DATA_PATH } from '../jetpack/extensions/shared/get-jetpack-data';
+
+const supportedJetpackBlocks = {
+	'contact-info': {
+		available: true,
+	},
+};
+
+const setJetpackData = ( {
+	isJetpackActive = false,
+	userData = null,
+	siteFragment = null,
+	blogId,
+} ) => {
+	const availableBlocks = supportedJetpackBlocks;
+	const jetpackEditorInitialState = {
+		available_blocks: availableBlocks,
+		jetpack: {
+			is_active: isJetpackActive,
+		},
+		siteFragment,
+		tracksUserData: userData,
+		wpcomBlogId: blogId,
+	};
+	global.window[ JETPACK_DATA_PATH ] = jetpackEditorInitialState;
```

Comment:
Just want to ask you @Tug about this line since it's hard to understand with the API calls temporarily disabled. Are we utilizing `global.window` here so jetpack code can access `jetpackEditorInitialState` when initializing, similar to how they're doing it on the web side in [jetpack/extensions/editor.js](https://github.com/Automattic/jetpack/blob/master/extensions/editor.js)?